src/index.html

```diff
@@ -20,8 +20,692 @@
+                <a
+                  href="#Photo"
+                  class="nav__link"
+                >
+                  Photo
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <main class="main">
+        <div
+          class="card"
+          data-qa="card-hover"
+        >
+          <div class="card__img">
+            <img
+              src="images/imac.jpeg"
+              alt="Apple iMac 27 Retina"
+            />
+          </div>
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__id">Product code: 195434</p>
+          <div class="card__review review">
+            <div class="review__stars">
+              <div class="stars stars--0">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--1">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--2">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--3">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="stars stars--5">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+            </div>
+            <div class="review__points">Reviews: 5</div>
```

Comment:
Use the semantic tag for text, fix it everywhere
```suggestion
            <p class="review__points">Reviews: 5</p>
```